export_report.py

```diff
@@ -0,0 +1,156 @@
+            m = p.match(f)
+            if 'reports' not in root and p.match(f) and m.group(2) == 'json':
+                df = pd.read_json(root+'/'+f, lines = True)
+
+                data = [dict(
+                type = 'scatter',
+                x = df['timestamp'],
+                y = df['latency']/1000000,
+                mode = 'markers',
+                transforms = [dict(
+                    type = 'groupby',
+                    groups = df['code']
+                )]
+                )]
+
+                layout = dict(
+                    title = '<b>Latency per Request: {}</b>'.format(m.group(1)),
+                    xaxis = dict(title = 'Time', showgrid = 'true', ticklabelmode = "period"),
+                    yaxis = dict(title = 'Milliseconds (log)', type = 'log'),
+                )
+
+                fig_dict = dict(data=data, layout=layout)
+
+                pio.write_image(fig_dict, m.group(1)+".png", engine="kaleido", width=1600, height=900, validate=False)
+                graphs[m.group(1)] = m.group(1)+".png"
+    return graphs
+
+
+def showgraphGraphs(file):
+    p = re.compile(r'[\w-]+_([\w-]+).(\w.+)')
+    m = p.match(file)
+    if p.match(file) and m.group(2) == 'json':
+        df = pd.read_json(file, lines = True)
+
+        data = [dict(
+        type = 'scatter',
+        x = df['timestamp'],
+        y = df['latency']/1000000,
+        mode = 'markers',
+        transforms = [dict(
+            type = 'groupby',
+            groups = df['code']
+        )]
+        )]
+
+        layout = dict(
+            title = '<b>Latency per Request: {}</b>'.format(m.group(1)),
+            xaxis = dict(title = 'Time', showgrid = 'true', ticklabelmode = "period"),
+            yaxis = dict(title = 'Milliseconds (log)', type = 'log'),
+        )
+
+        fig_dict = dict(data=data, layout=layout)
+
+        pio.show(fig_dict, engine="kaleido", width=1600, height=900, validate=False)
+
+def readReports():
+    reports = {}
+    for root, _, files in os.walk("."):
+        for f in files:
+            p = re.compile(r'[\w-]+_([\w-]+)-report-\d{4}-\d{2}-\d{2}.(\w.+)')
+            m = p.match(f)
+            if 'reports' in root and p.match(f) and m.group(2) == 'json':
+                df = pd.read_json(root+'/'+f, lines = True)
+
+                lat = df['latencies'][0]
+                reports[m.group(1)] = {
+                    'requests':     int(df['requests']),
+                    'rate':         float(df['rate']),
+                    'duration':     int(df['duration']),
+                    'min':          int(lat['min']),
+                    'mean':         int(lat['mean']),
+                    'max':          int(lat['max']),
+                    'success':      float(df['success']),
+                    'status_codes': df['status_codes'][0],
+                    'errors':       df['errors'][0],
+                }
+    return reports
+
+def writeDocx(reports, graphs):
+    date = datetime.datetime.now()
```

Comment:
It may be a good idea to use utcnow() instead of now() here. Otherwise, it will be timezone-dependent